src/main/scala/scorex/core/api/http/NodeViewApiRoute.scala

```diff
@@ -36,7 +36,7 @@ case class NodeViewApiRoute[P <: Proposition, TX <: Transaction[P]]
   type VL <: Vault[P, TX, PM, _ <: Vault[P, TX, PM, _]]
 
   //TODO null?
-  private val source: ConnectedPeer = null
+  //private val source: ConnectedPeer = null
```

Comment:
If `source` is not needed, maybe we shoudl removed this line completely, instead of just commenting it out?

Reply:
I think so